packages/compass-e2e-tests/tests/atlas-login.test.ts

```diff
@@ -41,9 +42,8 @@ describe('Atlas Login', function () {
   let stopMockAtlasServer: () => Promise<void>;
 
   before(async function () {
-    if (TEST_COMPASS_WEB) {
-      this.skip();
-    }
+    skipForWeb(this, 'atlas-login not supported in compass-web');
```

Comment:
There's still a couple of places where early returns are missing for `skipForWeb()` inside `before()` hooks :)

Reply:
you can call skip inside before hooks and it is implemented with a throw. you only have to early return in after hooks where skip is not available. Or am I misunderstanding you?

Reply:
Basically: The PR won't be green if we don't skip over bits that will break in compass web.